exchanges/localbitcoins/localbitcoins_wrapper.go

```diff
@@ -236,6 +236,7 @@ func (l *LocalBitcoins) UpdateOrderbook(p currency.Pair, assetType asset.Item) (
 	orderBook.Pair = p
 	orderBook.ExchangeName = l.Name
 	orderBook.AssetType = assetType
+	orderBook.NotAggregated = true
```

Comment:
if u can find people who r putting up limits on the excahnge pls tell them to contact me ill do some nice OTC deals with them
![image](https://user-images.githubusercontent.com/31364354/101437262-e1eec800-3963-11eb-9593-de16156c2c2e.png)